src/ui/server/helpers/sanitise-data/index.ts

```diff
@@ -40,7 +48,8 @@ const sanitiseValue = (value: string | number | boolean) => {
     return false;
   }
 
-  if (shouldChangeToNumber(value)) {
+  // should not change number if COMPANY_NUMBER or PHONE_NUMBER
+  if (shouldChangeToNumber(value) && key !== COMPANY_NUMBER && key !== PHONE_NUMBER) {
```

Comment:
Would be cleaner to have the additional checks in `shouldChangeToNumber` function, ie. if X field || Y field return false,

Can be done after this PR in the interests of getting this in

Reply:
will do after